### ISSUE TITLE
Stremio doesn't go fullscreen on secondary monitors

### ISSUE THREAD

**Issue by hyoretsu** [Stremio/stremio-bugs#466]
**Describe the bug**
If you have two monitors, and try to go fullscreen on your secondary one, the window will simply disappear. You can exit by pressing ESC. If you move Stremio to your main monitor and try again, it'll work.

**To Reproduce**
1. Go to any stream
2. Put Stremio in your secondary monitor
3. Go fullscreen

**Expected behavior**
It should go fullscreen on my secondary monitor too

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro 22H2

**Comment by bejzikkk** [comment]
yeah ,its a thing... only workaround is to mirror displays,at the moment

**Comment by TRtomasz** [comment]
Closing in favor of fixing this in stremio 5 new shell
